tensorboard/webapp/runs/store/runs_reducers_test.ts

```diff
@@ -158,6 +158,7 @@ describe('runs_reducers', () => {
             ],
           },
         },
+        expNameByExpId: {},
```

Comment:
QQ. If it's optional, don't we need to add this line?

Reply:
Oh, I have question. Is it an ok idea to make a field in the action, to be optional? I declared it as optional, but I am not sure whether it is an ok practice. wdyt?

Reply:
Optional action props should be fine conceptually and are [already used elsewhere](https://github.com/tensorflow/tensorboard/blob/d4c5c825a5677bf1d22d1d9290bb439f0ce7e861/tensorboard/webapp/runs/actions/runs_actions.ts#L90)

Reply:
Ok, removed empty dictionaries.